src/special_arrays.jl

```diff
@@ -111,6 +112,95 @@ function inverse_at!(x::AbstractVector, index, t::UnitVector, y::AbstractVector)
     index
 end
 
+####
+#### unit_vector_norm
+####
+
+struct UnitVectorNorm <: VectorTransform
+    n::Int
+    chi_prior::Bool
+    function UnitVectorNorm(n::Int; chi_prior::Bool = true)
+        @argcheck n ≥ 2 "Dimension should be at least 2."
+        new(n, chi_prior)
+    end
+end
+
+"""
+$(SIGNATURES)
+
+Transform `n ≥ 2` real numbers to a unit vector of length `n` and a radius, under the
+Euclidean norm.
```

Comment:
Should it document that the return value of the transform and argument of its inverse are the tuple `(unit_vector, norm)`?

Reply:
Thanks, I clarified this.